app/models/katello/content_view.rb

```diff
@@ -1,6 +1,8 @@
 module Katello
   # rubocop:disable Metrics/ClassLength
   class ContentView < Katello::Model
+    audited :associations => [:repositories, :environments]
```

Comment:
I don't quite like the way the audit messages are formed when:
1. creating a composite content view
`Added CompositeView to CompositeView`

2. adding a content view to the composite view
`Added N/A to CompositeView`

http://www.clipular.com/posts/6250077648322560?k=HwS3--b7nRKv1Y6sNyA-_BGrJFk